manifests/s/sharkdp/hexyl/0.16.0/sharkdp.hexyl.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created with komac v2.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: sharkdp.hexyl
+PackageVersion: 0.16.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: hexyl-v0.16.0-x86_64-pc-windows-gnu/hexyl.exe
+ReleaseDate: 2024-12-27
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/sharkdp/hexyl/releases/download/v0.16.0/hexyl-v0.16.0-x86_64-pc-windows-gnu.zip

```

```suggestion
  InstallerUrl: https://github.com/sharkdp/hexyl/releases/download/v0.16.0/hexyl-v0.16.0-x86_64-pc-windows-msvc.zip
```

After replacing this variants suppresses `Trojan:Script/Wacatac.B!ml` reports. But I don't have confidence is this a reasonable action...